aspnetcore/toc.yml

```diff
@@ -883,15 +883,15 @@
             - name: Overview
               uid: security/authentication/azure-active-directory/index
             - name: Integrate Azure AD into a web app
-              href: https://azure.microsoft.com/documentation/samples/active-directory-dotnet-webapp-openidconnect-aspnetcore/
+              href: https://github.com/Azure-Samples/active-directory-dotnet-webapp-openidconnect-aspnetcore
```

Comment:
This link has been archived. Consider replacing with the recommended link (https://github.com/Azure-Samples/ms-identity-aspnetcore-webapp-tutorial).